.gitignore

```
@@ -220,4 +220,12 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-
+# Python
+Python_Module/.eggs/
+Python_Module/MangDang/mini_pupper/__pycache__
+Python_Module/MangDang.egg-info/
+.pytest_cache
+Python_Module/ChangeLog
+Python_Module/AUTHORS
+mock_api/ChangeLog
+mock_api/AUTHORS
```

```suggestion
Python_Module/ChangeLog
Python_Module/AUTHORS
mock_api/ChangeLog
mock_api/AUTHORS
```

The Python cache file is already defined here, so there is no need to define it again.

https://github.com/mangdangroboticsclub/mini_pupper_bsp/pull/22/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R63-R88